mill-git/src/com/goyeau/mill/git/GitVersionModule.scala

```diff
@@ -35,19 +38,20 @@ object GitVersionModule extends ExternalModule {
                 if (isDirty) s"-${distance.toInt + 1}-${uncommitted()}"
                 else s"-$distance-${hash.take(hashLength)}$snapshotSuffix"
               }
-              s"$tag$distanceHash"
+              MillSuccess(s"$tag$distanceHash")
             case untaggedRegex(hash) =>
-              if (isDirty) uncommitted()
-              else s"${hash.take(hashLength)}$snapshotSuffix"
-            case _ => throw new IllegalStateException(s"Unexpected git describe output: $description")
```

Comment:
`./mill __.style` doesn't like `throw new...`, so changed this up to use pattern matching instead of `fold`